### PR TITLE
RAKwireless WisNode Documentation

### PR DESCRIPTION
**Pull request by kneelyrac**

### CODE REVIEW

File: docs/use-the-network/devices/ready-to-use/rakwireless/rak7201.mdx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    "img/use-the-network/devices/ready-to-use/rakwireless/rak7201/RAK7201.png"
+  )}
+/>
+
+
+## Overview
+
+The RAK7201 WisNode Button 4K is a remote wireless trigger device. It supports user-defined functions for each key and is based on the LoRaWAN 1.0.2 protocol. Supported LoRaWAN bands are IN865, EU868, AU915, US915, KR920, and AS923.
+
+This WisNode Button is suitable for a variety of Smart Home applications: entertainment system control, control of lights, a snooze button for the alarms, or a remote trigger. With long-range wireless connectivity, amazing battery life, and four programmable buttons, there are a ton of applications this device can fit into.
```
Comment (Perronef5) [inline docs/use-the-network/devices/ready-to-use/rakwireless/rak7201.mdx]: Hi @kneelyrac I suggest we change this paragraph a bit as its a bit hard to understand. I like using the https://hemingwayapp.com/ to help me come up with more bold and clear sentences :)

I attached a screenshot of what the hemminwayapp was saying.

Screen Shot 2022-06-13 at 11 35 44 AM


Below I think I was able to improve it a bit. Let me know what you think.

Comment (Perronef5) [inline docs/use-the-network/devices/ready-to-use/rakwireless/rak7201.mdx]:
```suggestion
This WisNode Button is suitable for a variety of Smart Home applications. Including, but not limited to, entertainment system control, control of lights, a snooze button for the alarms,  or a remote trigger. A device with long-range wireless connectivity, amazing battery life, and four programmable buttons. There are a tons of applications this device can fit into.
```

**Review by Perronef5 — COMMENTED** [review]
Left some comments

File: docs/use-the-network/devices/ready-to-use/rakwireless/rak7201.mdx (hunk 1 above)
Comment (kneelyrac) [reply docs/use-the-network/devices/ready-to-use/rakwireless/rak7201.mdx]: Thank you for your suggestion, @Perronef5. :)

**Comment by kneelyrac** [comment]
Hi, @Perronef5. 

I have requested a review of this PR. Let me know if I need to change something. 

Thank you!

File: sidebarsDocs.js [hunk 2]
```diff
@@ -1007,16 +1007,24 @@ module.exports = {
...
   ],
+
   rakwireless: [
     {
-      type: 'link',
-      label: '<- Back',
-      href: '/use-the-network/devices/ready-to-use',
+      type: "link",
+      label: "<- Back",
+      href: "/use-the-network/devices/ready-to-use",
```
Comment (samgutentag) [inline sidebarsDocs.js]:
```suggestion
      type: 'link',
      label: '<- Back',
      href: '/use-the-network/devices/ready-to-use',
```

Comment (samgutentag) [reply sidebarsDocs.js]: swapping double for single quotes

File: docs/use-the-network/devices/ready-to-use/rakwireless/rak7431.mdx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+- Mounting: Wall, DIN rail, and magnetic mounting.
+- Compliant with IEC61000-4-2, IEC61000-4-4, and 18kV HMB ESD protection.
+
+
+### Useful Resources
+
+- [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7431/Quickstart/)
+- [Datasheet](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7431/Datasheet/)
+- [AT Command Manual](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7431/AT-Command-Manual/)
+- [Store: RAK7431 WisNode Bridge Serial](https://store.rakwireless.com/products/rak7431?utm_source=RAK7431WisNodeBridge&utm_medium=Document&utm_campaign=BuyFromStore)
```
Comment (samgutentag) [inline docs/use-the-network/devices/ready-to-use/rakwireless/rak7431.mdx:55]:
```suggestion
- [Store: RAK7431 WisNode Bridge Serial](https://store.rakwireless.com/products/rak7431)
```

cleaning up links

File: docs/use-the-network/devices/ready-to-use/rakwireless/rak7431.mdx [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+## Connecting to Helium (The People's Network)
+
+:::tip
+Before going through each and every step in connecting the RAK7431 to Helium, it is recommended to check first our [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/rak7431/Quickstart/).
+:::
```
Comment (samgutentag) [inline docs/use-the-network/devices/ready-to-use/rakwireless/rak7431.mdx]:
```suggestion
:::tip

Before going through each and every step in connecting the RAK7431 to Helium, it is recommended to check first our [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/rak7431/Quickstart/).

:::
```

extra white space padding for formatting

File: docs/use-the-network/devices/ready-to-use/rakwireless/rak7201.mdx [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+- Working environment: Indoor
+
+
+### Useful Resources
+
+- [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7201/Quickstart)
+- [Datasheet](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7201/Datasheet/)
+- [AT Command Manual](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7201/AT-Command-Manual)
+- [Firmware Upgrade Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7201/Firmware-Upgrade-Guide/)
+- [RAK7201 WisNode Button 4K](https://store.rakwireless.com/products/wisnode-button-4k-rak7201?utm_source=RAK7201WisNodeButton4K&utm_medium=Document&utm_campaign=BuyFromStore)
```
Comment (samgutentag) [inline docs/use-the-network/devices/ready-to-use/rakwireless/rak7201.mdx:46]:
```suggestion
- [RAK7201 WisNode Button 4K](https://store.rakwireless.com/products/wisnode-button-4k-rak7201)
```

cleaning up links

File: docs/use-the-network/devices/ready-to-use/rakwireless/rak7201.mdx [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+
+
+## Connecting to Helium (The People's Network)
+
+:::tip
+Before going through each and every step in connecting the RAK7201 to Helium, it is recommended to check first our [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7201/Quickstart/).
+:::
```
Comment (samgutentag) [inline docs/use-the-network/devices/ready-to-use/rakwireless/rak7201.mdx]:
```suggestion
:::tip

Before going through each and every step in connecting the RAK7201 to Helium, it is recommended to check first our [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7201/Quickstart/).

:::
```

padding white space for formatting

File: docs/use-the-network/devices/ready-to-use/rakwireless/rak10700.mdx [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+- LoRa P2P support.
+- LoRa, BLE, and GPS antenna are integrated inside the casing.
+- Micro USB for configuration and charging of the battery.
+- Supports global license-free frequency band: RU864, IN865, EU868, RU864, US915, AU915, KR920, and AS923.
+
+### Useful Resources 
+- [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisBlock/RAK10700/Quickstart/)
+- [Datasheet](https://docs.rakwireless.com/Product-Categories/WisBlock/RAK10700/Datasheet/)
+- [AT Command Manual](https://docs.rakwireless.com/Product-Categories/WisBlock/RAK10700/AT-Command-Manual/)
+- [Store: RAK10700 WisBlock GNSS Tracker for LoRaWAN](https://store.rakwireless.com/products/lpwan-tracker-solution-kit?utm_source=RAK10700&utm_medium=Document&utm_campaign=BuyFromStore)
```
Comment (samgutentag) [inline docs/use-the-network/devices/ready-to-use/rakwireless/rak10700.mdx:51]:
```suggestion
- [Store: RAK10700 WisBlock GNSS Tracker for LoRaWAN](https://store.rakwireless.com/products/lpwan-tracker-solution-kit)
```

cleaning up links

File: docs/use-the-network/devices/ready-to-use/rakwireless/rak10700.mdx [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+- 4: EU433
+- 5: EU868
+- 6: KR920
+- 7: IN865
+- 8: US915
+- 9: AS923-2
+- 10: AS923-3
+- 11: AS923-4
+- 12: RU864
+:::
```
Comment (samgutentag) [inline docs/use-the-network/devices/ready-to-use/rakwireless/rak10700.mdx]:
```suggestion
:::info

The different regions are selected by number:

- 0: AS923
- 1: AU915
- 2: CN470
- 3: CN779
- 4: EU433
- 5: EU868
- 6: KR920
- 7: IN865
- 8: US915
- 9: AS923-2
- 10: AS923-3
- 11: AS923-4
- 12: RU864

:::
```

added white space padding for formatting

File: docs/use-the-network/devices/ready-to-use/rakwireless/rak2171.mdx [hunk 9]
```diff
@@ -0,0 +1,470 @@
...
+- [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK2171/Quickstart/)
+- [Datasheet](https://docs.rakwireless.com/Product-Categories/WisNode/RAK2171/Datasheet/)
+- [Store: RAK2171 WisNode TrackIt](https://store.rakwireless.com/products/wisnode-trackit-set-rak2171?utm_source=RAK2171WisNodeTrackIt&utm_medium=Document&utm_campaign=BuyFromStore)
+
+
+## WisNode TrackIt Helium Integration
+
+:::tip
+Before going through each and every step in connecting RAK2171 to Helium, it is recommended to check first the [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK2171/Quickstart/).
+:::
```
Comment (samgutentag) [inline docs/use-the-network/devices/ready-to-use/rakwireless/rak2171.mdx]:
```suggestion
:::tip

Before going through each and every step in connecting RAK2171 to Helium, it is recommended to check first the [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK2171/Quickstart/).

:::
```

whitespace padding for formatting

File: docs/use-the-network/devices/ready-to-use/rakwireless/rak2171.mdx [hunk 10]
```diff
@@ -0,0 +1,470 @@
...
+- Operating temperature: -20 °C to +60 °C
+- IP65 rating
+- Android and iOS application for easy management, configuration, and location tracking
+
+
+### Useful Resources
+
+- [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK2171/Quickstart/)
+- [Datasheet](https://docs.rakwireless.com/Product-Categories/WisNode/RAK2171/Datasheet/)
+- [Store: RAK2171 WisNode TrackIt](https://store.rakwireless.com/products/wisnode-trackit-set-rak2171?utm_source=RAK2171WisNodeTrackIt&utm_medium=Document&utm_campaign=BuyFromStore)
```
Comment (samgutentag) [inline docs/use-the-network/devices/ready-to-use/rakwireless/rak2171.mdx:43]:
```suggestion
- [Store: RAK2171 WisNode TrackIt](https://store.rakwireless.com/products/wisnode-trackit-set-rak2171)
```

cleaning up links

File: docs/use-the-network/devices/ready-to-use/rakwireless/rak2171.mdx [hunk 11]
```diff
@@ -0,0 +1,470 @@
...
+1. While the device is joining, you can create the integration. Select **Integrations** from the menu in the left panel.
+
+
+<img
+  src={useBaseUrl(
+    "img/use-the-network/devices/ready-to-use/rakwireless/rak2171/5.integration-page.png"
+  )}
+/>
+
+2. To create an integration, click on the **Add New Integration** button. A list of the available integrations will appear. As for this guide, click **Datacake**. For further information, refer to the [**Integrations** documentation](https://docs.helium.com/use-the-network/console/integrations/).
```
Comment (samgutentag) [inline docs/use-the-network/devices/ready-to-use/rakwireless/rak2171.mdx]:
```suggestion
2. To create an integration, click on the **Add New Integration** button. A list of the available integrations will appear. As for this guide, click **Datacake**. For further information, refer to the [**Integrations** documentation](/use-the-network/console/integrations/).
```

swap public URL for relative link

File: docs/use-the-network/devices/ready-to-use/rakwireless/rak7200.mdx [hunk 12]
```diff
@@ -0,0 +1,168 @@
...
+- Micro USB for configuration and charging of the battery.
+- Supports global license-free frequency band: IN865, EU868, US915, AU915, KR920, and AS923.
+
+
+### Useful Resources
+
+- [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7200/Quickstart/)
+- [Datasheet](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7200/Datasheet/)
+- [AT Command Manual](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7200/AT-Command-Manual)
+- [Store: RAK7200 WisNode Track Lite](https://store.rakwireless.com/products/rak7200-lora-tracker?utm_source=RAK7200WisNodeTrackLite&utm_medium=Document&utm_campaign=BuyFromStore)
```
Comment (samgutentag) [inline docs/use-the-network/devices/ready-to-use/rakwireless/rak7200.mdx:45]:
```suggestion
- [Store: RAK7200 WisNode Track Lite](https://store.rakwireless.com/products/rak7200-lora-tracker)
```

link cleanup

File: docs/use-the-network/devices/ready-to-use/rakwireless/rak7200.mdx [hunk 13]
```diff
@@ -0,0 +1,168 @@
...
+
+## Connecting to Helium (The People's Network)
+
+:::tip
+Before going through each and every step in connecting the RAK7200 to Helium, it is recommended to check first our [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7200/Quickstart/).
+:::
```
Comment (samgutentag) [inline docs/use-the-network/devices/ready-to-use/rakwireless/rak7200.mdx]:
```suggestion
:::tip

Before going through each and every step in connecting the RAK7200 to Helium, it is recommended to check first our [Quick Start Guide](https://docs.rakwireless.com/Product-Categories/WisNode/RAK7200/Quickstart/).

:::
```

whitespace padding for formatting

**Comment by samgutentag** [comment]
hi @kneelyrac these docs look great! made a few suggestions to clean up some links and do some whitespace formatting. If they look good to you I will do one more auto format pass and get these merged.  Thanks!

**Comment by kneelyrac** [comment]
> hi @kneelyrac these docs look great! made a few suggestions to clean up some links and do some whitespace formatting. If they look good to you I will do one more auto format pass and get these merged. Thanks!

Hello, @samgutentag. 

Thank you for the review! The whitespace formatting is good on my end. But as for the link, I'd like to use the product link with the `utm`  for SEO purposes.

**Comment by samgutentag** [comment]
Not a problem. Doing one final formatting pass with `prettier` and then ready to merge.  Thank you so much!

**Review by samgutentag — APPROVED** [review]

**Pull request merged**